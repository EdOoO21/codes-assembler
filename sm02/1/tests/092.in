57 156 126 143 621 205 -456 640 646 980 796 377 561 656 -807 167 70 -35 -675 716 700 -841 172 367 292 -307 -873 -911 765 -947 314 813 297 638 -663 766 190 -674 240 259 -679 489 -158 -12 522 382 597 -661 460 -757 380 774 947
