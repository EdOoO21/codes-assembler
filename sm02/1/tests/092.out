947
774
380
-757
460
-661
597
382
522
-12
-158
489
-679
259
240
-674
190
766
-663
638
297
813
314
-947
765
-911
-873
-307
292
367
172
-841
700
716
-675
-35
70
167
-807
656
561
377
796
980
646
640
-456
205
621
143
126
156
57
